docs/development.md

```diff
@@ -312,3 +325,15 @@ docker-compose restart worker
 docker-compose logs -f
 ```
 
+### Patching the environment
+
+Note that you can also update the environment in a service by modifying the
+`docker-compose.yml` file, and re-running:
+```
+docker-compose up -d
+```
+
+To check that the env. variable has been updated, run:
+```
+docker-compose exec <service> env
+```
```

Comment:
Again this is just a repeat of how to use docker-compose. We could hyperlink the docker-compose docs somewhere near the top if you think it's useful, otherwise I think we can assume people either already know this, or can google it.